ERCS/erc-7699.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 7699
+title: Soul Resonance Token
+description: this EIP defines an interface for Soul Resonance Tokens
+author: Mike Smith (@mike0x1024), Abe Johnson <abe@burve.io>, Alvis Kaiser <alvis@burve.io>, WurDst Tsao <wurst@burve.io>
+discussions-to: https://ethereum-magicians.org/t/1
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-28
+requires: 20
+---
+
+## Abstract
+
+This EIP proposes the following improvements to Soul Bond Tokens:
+Extensibility to [ERC-20](./erc-20.md).
+Token Minting and Burning while preserving non-transferability.
```

Comment:
Your abstract should layout the goal of the proposal, and give the reader a high level technical overview of how it accomplishes that goal. Yours is probably a bit light on the technical side.